src/app/models/feedback.py

```diff
@@ -0,0 +1,120 @@
+# import logging
+
+from sqlalchemy import func, text
+from sqlalchemy.engine import Result
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import aliased
+from sqlalchemy.sql.expression import select
+
+from src.app.views.response.feedback import Feedback as ResponseFeedback
+from src.app.views.response.feedback import FeedbackCount
+from src.core.database.models.feedback import DataModelPredictionFeedback as dbFeedback
+from src.core.database.models.player import Player as dbPlayer
+
+
+class Feedback:
+    def __init__(self, session: AsyncSession) -> None:
+        self.session = session
+        # self.logger = logging.getLogger("Feedback")
+
+    async def get_feedback(self, player_names: list[str]):
```

Comment:
note that we may not want this query to be exposed to everyone "feedback_text" & "label" can be considered "sensitivee"

Reply:
do we just take away the route and leave this here for now?

Reply:
for now that is good yeah